cli/src/main/java/io/hyperfoil/cli/commands/WrkAbstract.java

```diff
@@ -117,18 +117,21 @@ public void mainMethod(String[] args, Class wrkClass) {
 
    //   @CommandDefinition(name = "wrk", description = "Runs a workload simluation against one endpoint using the same vm")
    public abstract class AbstractWrkCommand implements Command<HyperfoilCommandInvocation> {
-      @Option(shortName = 'c', description = "Total number of HTTP connections to keep open", required = true)
+      @Option(shortName = 'c', description = "Total number of HTTP connections to keep open", defaultValue = "10")
       int connections;
 
-      @Option(shortName = 'd', description = "Duration of the test, e.g. 2s, 2m, 2h", required = true)
+      @Option(shortName = 'd', description = "Duration of the test, e.g. 2s, 2m, 2h", defaultValue = "10s")
       String duration;
 
-      @Option(shortName = 't', description = "Total number of threads to use.")
+      @Option(shortName = 't', description = "Total number of threads to use.", defaultValue = "2")
       int threads;
 
       @Option(shortName = 's', description = "!!!NOT SUPPORTED: LuaJIT script")
       String script;
 
+      @Option(shortName = 'p', description = "Periodic output of results during benchmark run", hasValue = false)
```

Comment:
Is this something specific to HF's implementation? If original wrk does not have it, we shouldn't either.

Reply:
The original wrk did not write out the throughput every second.  So we either remove that completely or add a flag to enable it as additional behaviour. This flag restores the default behaviour

Reply:
OK, I don't remember where this came from then. Please drop it completely.